### ISSUE TITLE
Rename "Vertical Sharding"

### ISSUE THREAD

**Issue by derekperkins** [youtube/vitess#2365]
The current sharding docs look like this:

![image](https://cloud.githubusercontent.com/assets/3588778/21124299/5304d6f6-c09b-11e6-8be8-6360c8f4e0b4.png)

Calling it "sharding" immediately makes your mind think horizontal shards, and "vertical" typically refers to instance sizing. That also makes you have to say "horizontal sharding", which probably makes people wonder if that is different than normal DB sharding. Eliminating vertical sharding in favor of something else would also let us just use "sharding" in a horizontal context.

Here are a few options to get the conversation started:

1. shard groups
1. data segmentation
1. keyspace separation
1. keyspace creation
1. keyspace segmentation
1. keyspace architecture

**Comment by alainjobart** [comment]
The wording could be better, I agree.

One thing to mention is that even though now we only do ''vertical splits", we plan to also do merges, and in general just table moves from one unsharded keyspace to another unsharded keyspace. At some point, we'll also support moving tables from sharded to sharded keyspaces. So the naming should reflect that, it can't be "splits".

So a few options to get the ball moving:

10. table(s) move
11. table(s) relocation
12. table(s) reorganization
13. keyspace shuffling
14. keyspace reorganization

**Comment by sougou** [comment]
I think these terms are ingrained in a lot of code and documentation. Instead, we should just provide good documentation what they mean.

I'll keep this bug open till it's documented.

**Comment by arthurnn** [comment]
Agree. I had multiple conversations with folks that didn't realize and/pr understood when I said sharding, meaning vertical and horizontal.

**Comment by sjmudd** [comment]
I've not used this feature (it sounds interesting) but to me the "vertical" sounds like "table relocation between keyspaces".  The table is not being re-organised (as that feels like something you do on the physical table itself).   If changing the documentation everywhere is complicated then just mentioning what this is in the main places where it is described may well be enough, then fix later.
